### PR TITLE
Enhance Player Throw

### PR DESCRIPTION
**Pull request by jordanmurray**
This pull request attempts to solve the problem of a player not being able to throw an object with much velocity. The fix simply adds calculations to the initial velocity of the thrown object. The initial velocity is also randomized to match behavior on official server.

### CODE REVIEW

**Comment by aramperes** [comment]
Hey, thank you for your contribution. May I ask where the values come from? Are they from trial-and-error?

**Comment by jordanmurray** [comment]
Yes, the values come from a process of trial-and-error to find the closest similar behavior to the official server.

File: src/main/java/net/glowstone/entity/GlowHumanEntity.java [hunk 1]
```diff
@@ -499,8 +500,15 @@ public GlowItem drop(ItemStack stack) {
 
         Location dropLocation = location.clone().add(0, getEyeHeight(true) - 0.3, 0);
         GlowItem dropItem = world.dropItem(dropLocation, stack);
-        Vector vel = location.getDirection().multiply(new Vector(0.11F, 0.0F, 0.11F));
-        vel.setY(0.006F);
+
+        Vector vel = location.getDirection().multiply(0.3);
+        ThreadLocalRandom tlr = ThreadLocalRandom.current();
+        double randOffset = 0.02;
```
Comment (Pr0methean) [inline src/main/java/net/glowstone/entity/GlowHumanEntity.java]: Extract these constants to private static final fields.

**Review by Pr0methean — CHANGES_REQUESTED** [review]

File: src/main/java/net/glowstone/entity/GlowHumanEntity.java (hunk 1 above)
Comment (aramperes) [reply src/main/java/net/glowstone/entity/GlowHumanEntity.java]: These values are approximate and have no real significance, I don't think constants are necessary. But it shouls have comments clarifying the calculations a bit.

**Review by Pr0methean — APPROVED** [review]

**Pull request merged**